Sources/PointFree/Admin/EpisodeCredit.swift

```diff
@@ -15,14 +15,22 @@ import Tuple
 import View
 import Views
 
-let showEpisodeCreditsMiddleware: Middleware<StatusLineOpen, ResponseEnded, Tuple1<User?>, Data> =
-  requireAdmin
-    <| writeStatus(.ok)
+let showEpisodeCreditsMiddleware: Middleware<
+  StatusLineOpen,
+  ResponseEnded,
+  Tuple1<User>,
```

Comment:
was able to make all the admin middlewares take an honest user instead of an optional, and also changed the formatting of these middleware decelerations to be a little more readable.